aldryn_forms/boilerplates/bootstrap3/templates/aldryn_forms/emails/notification.subject.txt

```diff
@@ -1 +1,3 @@
-[Form submission] {{ form_name }} 
\ No newline at end of file
+{% with subject=form_plugin.email_notification_subject %}
+{% if subject %}{{ subject }}{% else %}[Form submission] {{ form_name }}{% endif %}
+{% endwith %}
```

Comment:
@czpython I remember having problems with newlines in subjects because of [this protection](https://docs.djangoproject.com/en/1.8/topics/email/#preventing-header-injection). django-emailit just hides this detail and strips trailing newlines (at least), IIRC. But if we'll switch to plain Django functions and us e that templates - we'll get an exception.